debian-bullseye/verible.dockerfile

```diff
@@ -0,0 +1,52 @@
+# Authors:
+#   Unai Martinez-Corral
+#   Patrick Richer St-Onge
+#
+# Copyright 2022 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG REGISTRY='gcr.io/hdl-containers/debian/bullseye'
+
+#---
+
+FROM $REGISTRY/build/build AS build
+
+RUN apt-get update -qq \
+    && apt-get -y install --no-install-recommends \
+        gpg \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
You can do this just once, after the last time you install something (below).

Moreover, I would merge these three steps (gpg and bazel) in a single one. No need to have multiple layers.

Reply:
Right, thanks